GLAA.Web/Views/_ViewStart.cshtml

```diff
@@ -5,9 +5,9 @@
     var controller = ViewContext.RouteData.Values["Controller"];
     var controllerName = Convert.ToString(controller);
     string layoutFile;
-    switch (controllerName)
+    switch (controllerName.ToLower())
```

Comment:
This is even more sensible since we may be on a case sensitive OS!

Reply:
I was locally finding issues with this sometimes loading the admin layout, and sometimes not, this seemed to fix the issues for me.